Pensel/firmware/common.h

```diff
@@ -42,6 +42,10 @@
     #define min(v1, v2) (v1 < v2 ? v1 : v2)
 #endif
 
+#if !defined(NULL)
+    #define NULL ((void *) 0)
```

Comment:
Any reason for defining this yourself and not using `cstddef.h`?

Reply:
Didn’t know about this!